docs/internal-documentation/maintenance/StyleGuide.md

```diff
@@ -14,3 +14,13 @@ specified below.
 * For variable names in test case specifications each word is capitalized (e.g.
   "Domain Part"). If a word is an abbreviation, it is usually written with all
   capitals, e.g. "DNS".
+* When linking to other documents and you want to include the URL in the
+  rendered text, make sure to put angle brackets around the URL, i.e. use the
+  [CommonMark autolink] syntax.
+
+  If you forget the angle brackets the URL it is still render as a link on Github
+  (per [GFM autolink] syntax), but other tools (e.g. link checking tools) are
+  likely to not recognize it as a link.
```

Comment:
This kind of argument can be left out. It should be what, not why.

Reply:
When you apply the rule `what` is important, but when you change the rule `why` is important. I found the `why` to be non-obvious so I thought better to spell it out.

Reply:
OK, but could the "why" be shorter? E.g.
>If the angle brackets are not included, the URL it is render as a link on Github ([GFM autolink] syntax), but other tools may not recognize it as a link.